examples/pytorch_lightning/mae.py

```diff
@@ -114,5 +114,10 @@ def target_transform(t):
 
 accelerator = "gpu" if torch.cuda.is_available() else "cpu"
 
-trainer = pl.Trainer(max_epochs=10, devices=1, accelerator=accelerator)
+trainer = pl.Trainer(
+    max_epochs=10,
+    devices=1,
+    accelerator=accelerator,
+    strategy="ddp_find_unused_parameters_true",
```

Comment:
This example shouldn't be updated as it is for a single GPU and DDP doesn't make sense in that case. Could you revert this change?